.github/workflows/ci.yml

```diff
@@ -0,0 +1,20 @@
+name: CI
+on:
+  pull_request:
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2.1.0
+        with:
+          fetch-depth: 100
+      - name: Fetch tags
+        run: git fetch --depth=100 origin +refs/tags/*:refs/tags/*
```

Comment:
Do we really have to have access to the all tags in PR build?

Reply:
AFAIK this is actually an _optimization_, so that it doesn't download _everything_
it's important especially in big repos, but I figured it might as well be used everywhere, so that things are consisten